jax/_src/numpy/polynomial.py

```diff
@@ -292,6 +293,18 @@ def polymul(a1, a2, *, trim_leading_zeros=False):
   val = convolve(a1, a2, mode='full')
   return val
 
+@_wraps(np.polydiv, lax_description=_LEADING_ZEROS_DOC)
+def polydiv(a1, a2):
+  a1, a2 = _promote_dtypes_inexact(a1, a2)
+  m = len(a1) - 1
+  n = len(a2) - 1
+  scale = 1. / a2[0]
+  q = zeros(builtins.max(m - n + 1, 1))
```

Comment:
I don't think `builtins` here is required, as `max` is not redefined in this file.